src/Categories/Category/Monoidal/Interchange.agda

```diff
@@ -45,8 +34,8 @@ private
 -- monoidal functor.  (See also Categories.Functor.Monoidal.Tensor.)
 
 record HasInterchange (M : Monoidal C) : Set (o ⊔ ℓ ⊔ e) where
```

Comment:
I'm now tempted to add two more records: `HasBraidedInterchange` and `HasSymmetricInterchange` with the extra fields that are in the respective modules. This would completely separate the axioms from the proofs, so that the former could be used in down-stream proofs/constructions without pulling in the big concrete proof terms. But maybe this is for a future PR.